src/Microsoft.DotNet.Watcher.Core/Internal/Implementation/ProjectProvider.cs

```diff
@@ -40,10 +40,18 @@ private bool TryGetProject(string projectFile, out ProjectModel.Project project,
         {
             try
             {
-                var errors = new List<DiagnosticMessage>();
-                if (!ProjectReader.TryGetProject(projectFile, out project, errors))
+                if (!ProjectReader.TryGetProject(projectFile, out project))
                 {
-                    errorMessage = string.Join(Environment.NewLine, errors.Select(e => e.ToString()));
+                    if (project != null &&
+                        project.Diagnostics != null &&
```

Comment:
This can be simplified to `project?.Diagnostics != null` && `project.Diagnostics.Any()`